### PR TITLE
Add authController

### PR DESCRIPTION
**Pull request by darenang44**

### CODE REVIEW

File: client/app/auth/authController.js [hunk 1]
```diff
@@ -1,5 +1,16 @@
-angular.module('throwcast.auth', [])
+angular.module('throwcast.auth', ['ngRoute'])
 
+.config(function ($routeProvider) {
```
Comment (dting) [inline client/app/auth/authController.js]: The config really doesn't have anything to do with the auth controller.

I would move the config and the module creation and dependency declaration into its own file (probably `auth.js` in the same folder). Then use `angular.module('throwcast.auth')` to get the created module in this file.

File: client/app/app.js [hunk 2]
```diff
@@ -42,13 +34,13 @@ angular.module('throwcast', [
...
-.run(function ($rootScope, $location, authService) {
-  $rootScope.$on('$routeChangeStart', function (evt, next, current) {
-    if (next.$$route && next.$$route.authenticate && !authService.isAuth()) {
-      //evt.preventDefault(); commented out for test purposes
-      $location.path('/signin');
-    }
-  });
 });
+
+// .run(function ($rootScope, $location, authService) {
```
Comment (dting) [inline client/app/app.js]: Don't leave commented out code in the repository.

File: client/app/auth/authController.js (hunk 1 above)
Comment (darenang44) [inline client/app/auth/authController.js]: Done

File: client/app/app.js (hunk 2 above)
Comment (darenang44) [inline client/app/app.js]: Uncommented the code

**Comment by dting** [comment]
LGTM

**Pull request merged**